libcxx/include/cmath

```diff
@@ -315,6 +315,7 @@ constexpr long double lerp(long double a, long double b, long double t) noexcept
 #include <limits>
 #include <version>
 
+#include <__math/special_functions.h>
```

Comment:
As mentioned in our call the synopsis in this file needs to list the added hermite functions.
These functions also need to be uncommented in https://github.com/llvm/llvm-project/blob/main/libcxx/modules/std/cmath.inc

Reply:
6b28fc59cc674f15075ec058ac6e3b1b217747bc: uncommented in cmath.inc
58ff841509fefa04784d788af4b8e6f18c84d5dd: added to synopsis in cmath